### ISSUE TITLE
Setup inspiration and maybe time estimates?

### ISSUE THREAD

**Issue by Simmuh** [madMAx43v3r/chia-plotter#163]
Hello,

I plan on creating following plotting machine:

i9 9900k
128gb ddr4 ram with 3200mhz
2 SAS Drives 10k rpm 300gb

Using the SAS drives in raid 0 for temp 1 and the ram as ram disk for temp 2.

How fast do you reckon it will go? Should go under 20min per plot?

**Comment by madMAx43v3r** [comment]
no CPU is too slow for that, more like ~35 min

**Comment by madMAx43v3r** [comment]
yeah I think a cheap NVMe is faster than two 10k drives

**Comment by selcukercin** [comment]
Hello there what about ,

i9 10850k
48gb ddr4 ram with 2400mhz
1x 2b sanddisk extreme nvme ?

**Comment by Simmuh** [comment]
Yeah maybe I will use a firecuda 510 1tb or less with it instead. They have great tbw and for this short duration speed should be enough I think.

**Comment by selcukercin** [comment]
> Yeah maybe I will use a firecuda 510 1tb or less with it instead. They have great tbw and for this short duration speed should be enough I think.

Can you update your results after your test please ?
Thank you

**Comment by Simmuh** [comment]
> no CPU is too slow for that, more like ~35 min

Something that also came to mind is the following:

When will cpu and ram hit deminishing returns?

I don't think that when you throw 45 instead of 12 cores at it that it will even get 20% increase.

Also is ddr3 vs ddr4 even relevant as well? I have so many questions in relations to the scalability and cap

**Comment by madMAx43v3r** [comment]
people have tried with 128 cores and it did scale, maybe not 1:1, but it was faster
